sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Metrics/Metrics.cs

```diff
@@ -80,7 +80,12 @@ protected override bool IsFunction(SyntaxNode node)
 
         public override int GetCognitiveComplexity(SyntaxNode node)
         {
-            return 0;
+            return 0; // Not implemented
+        }
+
+        public override int GetExecutableLinesCount()
+        {
+            return 0; // Not implemented

```

Comment:
Could you double check with someone on SonarQube side this is the expected behavior?

Reply:
Didn't get the response back yet, will chase it up on Monday.